check-branch-name/action.yml

```
@@ -0,0 +1,22 @@
+name: "Check if the branch name follows our naming convention."
+description: "Action that checks if the name of the branch is following the our branch naming convention."
+
+runs:
+  using: composite
+  steps:
+    - uses: jungwinter/split@master
+      id: split
+      with:
+        msg: ${{ env.BRANCH_NAME }}
+        separator: "/"
+        maxsplit: -1
+
+    - name: Check branch name has slash
+      shell: bash
+      if: contains('/', env.BRANCH_NAME) == false
+      run: echo "[ERROR] Branch name ${{ env.BRANCH_NAME }} does not contain a prefix and a backslash." && exit 1
+
+    - name: Check branch prefix naming
+      shell: bash
+      if: contains(fromJson('["fix", "feat", "junk", "maint", "doc", "no-ci", "testing", "release"]'), steps.split.outputs._0) == false
```

@AlejandroFernandezLuces @jorgepiloto Isn't `test` accepted for the branch name?
If we look for `test` then both `test` and `testing` will work.
```suggestion
      if: contains(fromJson('["fix", "feat", "junk", "maint", "doc", "no-ci", "test", "release"]'), steps.split.outputs._0) == false
```